src/sass/components/footer.scss

```diff
@@ -0,0 +1,35 @@
+.footer-wrapper {  
+    display: flex;   
+    justify-content: space-between;
+    align-items: center;
+    @include border();    
+    display: flex;    
+    min-height: 50px;
+    background-color: $color-primary;    
+    padding: 0.4rem; 
+    @include title();
```

Comment:
Also, the mixin's names are not good.  The name should be more detailed. For example, you named it as 'title'. Which title is it? Does it the header or table title? You should rename it :)